SQL/advanced.sql

```diff
@@ -376,3 +376,32 @@ CREATE VIEW booking_costs_today AS (
   ORDER BY
     "cost" DESC
 );
+
+
+----------------------------------------------------------------------------------------
+/*
+STORED FUNCTIONS
+*/
+
+-- passing parameters
+CREATE OR REPLACE PROCEDURE remove_member(
+  member_id integer
+)
+AS $$
+BEGIN
+  DELETE FROM members WHERE memid = member_id;
+  IF NOT FOUND THEN
```

Comment:
correct!

do not forget that PL/pgSQL gives you more tools and features comparing to just SQL, it makes sense to use it,

Reply:
OK, got it